src/js/utils/browser.ts

```diff
@@ -36,6 +36,10 @@ export function isTizen(): boolean {
     return userAgentMatch(/SMART-TV/);
 }
 
+export function isTizenApp(): boolean {
+    return isTizen() && userAgentMatch(/Version\//);
```

Comment:
Could this be made a little more specific?

Reply:
It looks like the only difference between the browser and web application user agent is the inclusion of `Version` and it looks like the web app doesn't have `SamsungBrowser`. I think that might be as specific as we can get.